### PR TITLE
Need to upgrade the collections to 11.0

### PR DESCRIPTION
**Pull request by PPrinsloo**
I'm upgrading rating engine to Laravel 11 and this dependency is creating a conflict.

### CODE REVIEW

**Review by erikgaal — APPROVED** [review]
Can we extend the range instead?

**Comment by PPrinsloo** [comment]
> Can we extend the range instead?

so rather `>=10.0`?

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php": "^8.2",
-        "illuminate/collections": "^10.0",
+        "illuminate/collections": "^11.0",
```
Comment (PPrinsloo) [inline composer.json:21]:
```suggestion
        "illuminate/collections": ">=10.0 <12.0",
```

Comment (erikgaal) [inline composer.json]:
```suggestion
        "illuminate/collections": "^10 || ^11",
```

😃

**Review by erikgaal — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (PPrinsloo) [reply composer.json]: clicked the wrong button. I'll add that yes

**Review by PPrinsloo — COMMENTED** [review]
add range

File: composer.json (hunk 1 above)
Comment (PPrinsloo) [inline composer.json]:
```suggestion
        "illuminate/collections": "^10 || ^11",
```

**Review by erikgaal — APPROVED** [review]

**Pull request merged**